# HG changeset patch
# User Christopher Meng <cickumqt@gmail.com>
# Date 1398311299 0
# Branch cicku/rl_settingspy-edited-adding-paths-on-fed-1398311278921
# Node ID 78b616440f922f507747d9db90b927b912f40f0b
# Parent  4df3c1696c1beee18f8233ce1dfe39f1f7c03391
rl_settings.py edited, adding paths on Fedora

diff --git a/src/reportlab/rl_settings.py b/src/reportlab/rl_settings.py
--- a/src/reportlab/rl_settings.py
+++ b/src/reportlab/rl_settings.py
@@ -114,6 +114,7 @@
                 '/usr/local/Acrobat6/Resource/Font',    #Linux, Acrobat 5?
                 '/usr/local/Acrobat5/Resource/Font',    #Linux, Acrobat 5?
                 '/usr/local/Acrobat4/Resource/Font',    #Linux, Acrobat 4
+                '/usr/share/fonts/default/Type1'        #Linux, Fedora
                 '%(REPORTLAB_DIR)s/fonts',              #special
                 '%(REPORTLAB_DIR)s/../fonts',           #special
                 '%(REPORTLAB_DIR)s/../../fonts',        #special
@@ -130,6 +131,8 @@
                 'c:/windows/fonts',
                 '/usr/lib/X11/fonts/TrueType/',
                 '/usr/share/fonts/truetype',
+                '/usr/share/fonts'              #Linux, Fedora
+                '/usr/share/fonts/dejavu'       #Linux, Fedora
                 '%(REPORTLAB_DIR)s/fonts',      #special
                 '%(REPORTLAB_DIR)s/../fonts',   #special
                 '%(REPORTLAB_DIR)s/../../fonts',#special
